docs/controls/ImageEx.md

```diff
@@ -43,9 +43,9 @@ The [ImageEx](/dotnet/api/microsoft.toolkit.uwp.ui.controls.imageex) control dow
 
 ImageEx control supports use of Progress Indicator. This can be enabled by adding ImageEx template from previous release of the control.
 
-- [ImageEx Control with Progress Indicator XAML File](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/rel/7.0.0/Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageEx.xaml) is the XAML template used in v.7.0.0 of toolkit.
+- [ImageEx Control with Progress Indicator XAML File](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/rel/2.2.0/Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageEx.xaml) is the XAML template used in v.2.2.0 of toolkit.
 
-- [ImageEx Control XAML File](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/rel/7.0.0/Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageEx.xaml) is the XAML template used in the toolkit for the default styling.
+- [ImageEx Control XAML File](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/main/Microsoft.Toolkit.Uwp.UI.Controls.Core/ImageEx/ImageEx.xaml) is the XAML template used in the toolkit for the default styling.
```

Comment:
```suggestion
- [ImageEx Control XAML File](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/rel/7.0.0/Microsoft.Toolkit.Uwp.UI.Controls.Core/ImageEx/ImageEx.xaml) is the XAML template used in the toolkit for the default styling.
```

We still want to point to a specific version so when we move things in the future (if we do) the links don't break. These links were just missed in the last refactor so were still broken.